src/osdc/CMakeLists.txt

```diff
@@ -1,11 +1,9 @@
 set(osdc_files
-  Objecter.cc
-  Filer.cc)
-set(osdc_rbd_files
+  Filer.cc
```

Comment:
I may be misunderstanding the intent of this code but I believe there was no reason to break these `osdc_rbd_files` into a separate group which is why I merged them. Nothing else was referring to `osdc_rbd_files`.